charts/ocis/values.yaml

```diff
@@ -85,34 +85,37 @@ insecure:
   ocisHttpApiInsecure: false
 
 cache:
-  # -- Type of the cache to use. It defaults to no cache or a in memory cache depending on the service.
+  # -- Type of the cache to use. It defaults to no cache.
   # To disable the cache, set to "noop".
-  # Can be set to "redis", "redis-sentinel" or "etcd", then the address of Redis (Sentinel) / etcd node(s) needs to be set to `cache.nodes`.
-  type: ""
+  # Can be set to "redis-sentinel", "etcd" or "nats-js-kv" then the address of Redis (Sentinel) / etcd / nats-js node(s) needs to be set to `cache.nodes`.
+  type: "nats-js-kv"
   # -- Nodes of the cache to use.
-  nodes: []
-  # nodes:
-  #   - redis-master-1.ocis-redis.svc.cluster.local:6379
-  #   - redis-master-2.ocis-redis.svc.cluster.local:6379
+  nodes:
+    - "{{ .appNameNats }}:9233"
+  # -- Username to use for authentication
+  authusername: ""
 
 store:
   # -- Configure the store type. To disable the cache, set to "noop".
   # Defaults to "nats-js". Therefore `store.nodes` defaults to the address of the builtin NATS.
-  # Can be set to "redis", "redis-sentinel" or "etcd", then the address of Redis (Sentinel) / etcd node(s) needs to be set to `cache.nodes`.
-  type: nats-js
+  # Can be set to "redis-sentinel", "etcd" or "nats-js-kv" then the address of Redis (Sentinel) / etcd / nats-js node(s) needs to be set to `cache.nodes`.
+  type: nats-js-kv
   # -- Nodes of the store to use.
   nodes:
     - "{{ .appNameNats }}:9233"
+  # -- Username to use for authentication
+  authusername: ""
 
 registry:
   # -- Configure the service registry type.
   # Defaults to "kubernetes".
   # Can be set to "etcd" or "natsjs", then the address of etcd / nats node(s) needs to be set to `registry.nodes`.
```

Comment:
`Defaults to "kubernetes"` can be completely removed. It's wrong and we have a default for type, so we don't need to duplicate it in the comment.

Reply:
removed :+1: